### PR TITLE
#21 CSS System Setup

### PR DESCRIPTION
**Pull request by Cleop**
#21 - CSS System Setup Epic

Worked on issues:
#26 Created presets for Font sizes, colours, line heights, weights
#27 Customised values for spacing and border radius
#28 Customised values for max width and margin

I've kept some pixel values in comments on the css files because I think this will be helpful throughout the process so we don't have to keep on retranslating/calculating what Noga's values are vs. what our tachyons rem values are.

### CODE REVIEW

File: public/stylesheets/style.min.css [hunk 1]
```diff
@@ -1 +1,6298 @@
...-m{margin-right:var(--spacing-medium)}.mr4-m{margin-right:var(--spacing-large)}.mr5-m{margin-right:var(--spacing-extra-large)}.mr6-m{margin-right:var(--spacing-extra-extra-large)}.mr7-m{margin-right:var(--spacing-extra-extra-extra-large)}.mb0-m{margin-bottom:var(--spacing-none)}.mb1-m{margin-bottom:var(--spacing-extra-small)}.mb2-m{margin-bottom:var(--spacing-small)}.mb3-m{margin-bottom:var(--spacing-medium)}.mb4-m{margin-bottom:var(--spacing-large)}.mb5-m{margin-bottom:var(--spacing-extra-large)}.mb6-m{margin-bottom:var(--spacing-extra-extra-large)}.mb7-m{margin-bottom:var(--spacing-extra-extra-extra-large)}.mt0-m{margin-top:var(--spacing-none)}.mt1-m{margin-top:var(--spacing-extra-small)}.mt2-m{margin-top:var(--spacing-small)}.mt3-m{margin-top:var(--spacing-medium)}.mt4-m{margin-top:var(--spacing-large)}.mt5-m{margin-top:var(--spacing-extra-large)}.mt6-m{margin-top:var(--spacing-extra-extra-large)}.mt7-m{margin-top:var(--spacing-extra-extra-extra-large)}.mv0-m{margin-top:var(--spacing-none);margin-bottom:var(--spacing-none)}.mv1-m{margin-top:var(--spacing-extra-small);margin-bottom:var(--spacing-extra-small)}.mv2-m{margin-top:var(--spacing-small);margin-bottom:var(--spacing-small
```
Comment (nelsonic) [inline public/stylesheets/style.min.css]: is this _all_ of Tachyons...?

File: views/layout.hbs [hunk 2]
```diff
@@ -11,6 +11,7 @@
       gtag('config', 'UA-116110951-1');
     </script>
     <title>{{title}}</title>
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700|Nunito+Sans:400,700" rel="stylesheet">
```
Comment (nelsonic) [inline views/layout.hbs]: @Cleop is CharterIndex using "Nunito Sans" ...? 🤔

Comment (Cleop) [reply views/layout.hbs]: Yeah, smaller text is Nunito Sans in the style guide and large titles etc are Montserrat 👍

File: public/stylesheets/style.min.css (hunk 1 above)
Comment (Cleop) [reply public/stylesheets/style.min.css]: Yes I believe so. This is following the method to use the Tachyons-custom repo: https://github.com/tachyons-css/tachyons-custom . You access tachyons via the node module and then you overwrite/customise sections of it with your own code and then it's all compiled into CSS and minified.  I think this is the method required in order to customise tachyons. It's also the method that's been used here: https://github.com/InFact-coop/Stimmy-things

**Pull request merged**